ui_tests/src/test/resources/com/saucedemo/account/signInNegative.feature

```diff
@@ -0,0 +1,66 @@
+Feature: Sign In
+
+
+  @signIn
+
+  Scenario: Sign in with empty username and password
+    Given  I open start page
+    When  I set '' as username on Sign in page
+    And   I set ' ' as password on Sign in page
```

Comment:
@MAKELAQA As per the scenario name you are going to try to sign in with empty password, but the step for setting password is passing whitespace (` `)  as password i.e. it's not empty.
Please update the step

Reply:
regarding this one should I put (       ) for empty password or '      ' ?
please advise

Reply:
THIS IS WHAT I HAVE NOW AND FROM BEGINNING :
 I set ' ' as password on Sign in page

Reply:
Hey Yaroslav are We all good for tonight class ?